tools/webpack

```diff
@@ -46,6 +49,17 @@ def build_for_dev_server(host, port, minify, disable_host_check):
     if disable_host_check:
         webpack_args.append('--disable-host-check')
 
+    cwd = os.getcwd()
+    inotify_broken = False
+    with open('/proc/self/mounts') as mounts:
+        for line in mounts:
+            fields = line.split()
+            if fields[1] == cwd:
+                inotify_broken = fields[2] in ["nfs", "vboxsf"]
+    if inotify_broken:
+        webpack_args.append('--watch-poll=1000')
+
+    # TODO: Do something else here if inotify_broken.
```

Comment:
What would something else be here?  This comment is puzzling.

Reply:
Fall back to periodic polling. (Right now editing webpack_config.ts from the host doesn’t trigger a recompile in a VirtualBox guest at all.)